rabbitmq/configure.sls

```diff
@@ -0,0 +1,128 @@
+#!pyobjects
```

Comment:
The reason for it being done as pyobjects is because of the calculations necessary for determining the ram and disk limit values. I'm not sure how I would go about importing the results of the computation from a pyobjects file to a jinja/yaml file.